Web/starter_project_vanguard/pages/blog/blogs.tsx

```diff
@@ -0,0 +1,52 @@
+import { map } from 'cypress/types/bluebird'
+import React from 'react'
+import Blog from '@/components/blog/Blog'
+import Search from '@/components/blog/Search'
+
+import { useState } from 'react';
+import Pagination from '@/components/common/Pagination';
+
+
+function Blogs() {
+
+  const blogs = [
+    <Blog/>,
```

Comment:
is there a better way to iterate over a number of blogs in javascript, rather than manually writing the components. for example if the number of blogs is 50, or 100, we have to write 50, 100 components in this array.